homeassistant/components/asuswrt/bridge.py

```diff
@@ -52,6 +54,28 @@ def _get_dict(keys: list, values: list) -> dict[str, Any]:
     return dict(zip(keys, values))
 
 
+def handle_errors_and_zip(exceptions: Any, keys: list | None) -> Callable:
+    """Run library methods and zip results or manage exceptions."""
+
+    def _handle_errors_and_zip(func: Callable) -> Callable:
+        """Run library methods and zip results or manage exceptions."""
+
+        @functools.wraps(func)
+        async def _wrapper(self: Any, *args: Any, **kwargs: Any) -> dict[str, Any]:
+            try:
+                data = await func(self, *args, **kwargs)
```

Comment:
All the functions where you apply the decorator don't have any arguments besides the self. We can therefore remove args and kwargs